MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -7,11 +7,13 @@
 #import "MSMobileCenterInternal.h"
 #import "MSUtility+Application.h"
 
+static NSString *const kMSCustomSelectorPrefix = @"custom_";
 static NSString *const kMSReturnedValueSelectorPart = @"returnedValue:";
 static NSString *const kMSIsSwizzlingEnabledKey = @"MSAppDelegateForwarderEnabled";
 
 static NSHashTable<id<MSAppDelegate>> *_delegates = nil;
 static NSMutableSet<NSString *> *_selectorsToSwizzle = nil;
+static NSArray<NSString *> *_selectorsNotToOverride = nil;
```

Comment:
It's a bit more explicit, it's not only original selector but it's also the ones that we must not override by swizzling.